### ISSUE TITLE
Problem of training phrase

### ISSUE THREAD

**Issue by ansj11** [xanderchf/MonoDepth-FPN-PyTorch#6]
I am training your code on the Stanford 2D-3D-S dataset, but I got the predict depth map between 0 and 1, but the real depth can reach to 128, Do you know why?

**Comment by ansj11** [comment]
Otherwise, My training loss is difficult to reduce, always near 0.88
